lib/cryptoexchange/exchanges/bybit/services/contract_stat.rb

```diff
@@ -0,0 +1,51 @@
+module Cryptoexchange::Exchanges
```

Comment:
this is a scenario where `open_interest` and `index` data came from the same `url`